images/win/scripts/SoftwareReport/SoftwareReport.Generator.ps1

```diff
@@ -168,6 +165,13 @@ $markdown += New-MDHeader "Databases" -Level 3
 $markdown += Build-DatabasesMarkdown
 $markdown += New-MDNewLine
 
+$markdown += New-MDHeader "Database tools" -Level 3
```

Comment:
What is the difference between `Databases` and `Databases tools`?

Reply:
Databases represent database engines installed and available locally, database tools represent client tools that allows to communicate with remote data engines.

Reply:
Sounds good